client/package.json

```diff
@@ -351,6 +351,12 @@
         "icon": "$(add)",
         "description": "Start watching or suggesting a recipe in bitbake commands."
       },
+      {
+        "command": "bitbake.drop-all-recipes",
+        "title": "BitBake: Drop all recipe from the active workspace",
+        "description": "Clear all recipes in the recipes view and from all commands suggestions.",
+        "icon": "$(close)"
```

Comment:
I don't think we can do submenues on the title bar. The button is not too big or near anything sensitive.
It's not a big deal to click it accidentally anyway since it's just a feature of the most recently used recipes.

Reply:
if it is not a 'risky' operation then it is fine.

besides, it is rather simple to add a submenu by just adding the `submenu` property without the `command`:
![image](https://github.com/yoctoproject/vscode-bitbake/assets/47988425/d0bc8b2a-7743-46fd-8ae3-b607d97ad714)

Reply:
Ok I totally forgot about that menu possibility!
I don't think it's needed as explanied. This is just a temporary view with no significant data.